src/components/trip_view/cal/custom_cal_views/ToDoView.js

```diff
@@ -80,4 +80,9 @@ ToDoList.title = () => {
   return 'Trip To-Dos';
 };
 
+// This is required for react-big-calendar
+ToDoList.navigate = () => {
```

Comment:
This seems strange. This variable must be assigned for react-big-calendar?

Reply:
Can confirm—otherwise, it tries to call the navigate function at certain times and throws a null pointer exception. Admittedly, the likelihood of it attempting to call something is very low now (since the buttons to go between different days are gone), but I wanted to be safe! This prevents the crash you showed me in the lab when navigating with Back and Next in the todo list.